server/settings.go

```diff
@@ -31,21 +50,24 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 	if connection.Settings == nil {
 		connection.Settings = &ConnectionSettings{}
 	}
-	connection.Settings.Notifications = value
+	if !connection.sendNotification(args[1], value) {
+		return p.responsef(header, helpText)
+	}
```

Comment:
It's a little confusing how `sendNotification` mutates the setting state, and is at the same time used as a boolean control mechanism here. Also the name `sendNotification` sounds like it should be sending a notification, but that's not what's happening here